execute/table.go

```diff
@@ -1192,6 +1192,10 @@ func (b *ColListTableBuilder) SetValue(i, j int, v values.Value) error {
 }
 
 func (b *ColListTableBuilder) AppendValue(j int, v values.Value) error {
+	if v.IsNull() {
```

Comment:
This is so useful and safety, I want to propose making AppendBool, AppendInt, etc.  all private functions, and forcing us to use  AppendValue(j, values.New(false)), etc.

Reply:
I suspect we'll still want to expose the methods that accept raw types for performance reasons, since `values.Value` uses `interface{}` and reflection under the hood.  Were it not for this I would totally agree with you.